source/index-management.rst

```diff
@@ -71,7 +71,18 @@ All drivers MUST offer at least one of the sections of operations, the Standard
 Operation Parameters
 --------------------
 
-All drivers MUST include the specified parameters in each operation, with the exception of the options parameter which is OPTIONAL. As of 3.4 (see https://jira.mongodb.org/browse/SERVER-769) the server validates options passed to the createIndexes command -- drivers should be aware when testing that passing arbitrary options when the driver does not validate them could fail on the server.
+All drivers MUST include the specified parameters in each operation. This does not preclude a driver from offering more. A driver SHOULD NOT require a user to specify the options parameter if they wish to use the server defaults.
+
+As of 3.4 (see https://jira.mongodb.org/browse/SERVER-769) the server validates options passed to the createIndexes command -- drivers should be aware when testing that passing arbitrary options when the driver does not validate them could fail on the server.
+
+Deviations
+**********
+
+A non-exhaustive list of acceptable deviations are as follows:
+
+* Using named parameters in place of an options hash or class. For instance, ``collection.createIndex({x: 1}, commitQuorum: "all")``.
+
+* When using an ``Options`` class, if multiple ``Options`` classes are structurally equatable, it is permissible to consolidate them into one with a clear name. For instance, it would be permissible to use the name ``CreateIndexOptions`` as the options for ``createIndex`` and ``createIndexes``.
```

Comment:
I feel like we're getting overly prescriptive here, why do we need to clarify this?